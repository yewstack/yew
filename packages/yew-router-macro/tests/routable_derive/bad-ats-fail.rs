#[derive(yew_router::Routable)]
enum Routes {
    One,
}

#[derive(yew_router::Routable)]
enum RoutesTwo {
    #[at("/one")]
    #[at("/two")]
    One,
}

fn main() {}
