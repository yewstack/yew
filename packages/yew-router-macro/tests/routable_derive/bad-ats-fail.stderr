error: at attribute must be present on every variant
 --> $DIR/bad-ats-fail.rs:3:5
  |
3 |     One,
  |     ^^^

error: only one at attribute must be present
 --> $DIR/bad-ats-fail.rs:8:5
  |
8 | /     #[at("/one")]
9 | |     #[at("/two")]
  | |_________________^
